### PR TITLE
fix: Export `'channels'` as part of environments' export

### PR DESCRIPTION
**Pull request by mathbunnyru**
If I run the following:
```
mamba env create --name empty
mamba env export --name empty
```

it will give me:
```
name: empty
channels:
dependencies:

prefix: "/Users/asalikhov/test-mamba-root/envs/empty"
```

Note: `channels`, `dependencies` are always printed, even if empty.

But if I run `export` with `--json`, it will not print `channels:`:
```
{
  "dependencies": [
  ],
  "name": "empty",
  "prefix": "/Users/asalikhov/test-mamba-root/envs/empty"
}
```

I think it's always good to print `channels`, it makes parsing easier.

Related: https://github.com/mamba-org/mamba/issues/3582

### CODE REVIEW

File: micromamba/tests/test_env.py [hunk 1]
```diff
@@ -70,31 +70,47 @@ def export_env():
...
+
+
 @pytest.mark.parametrize("channel_subdir_flag", [None, "--channel-subdir"])
 @pytest.mark.parametrize("md5_flag", [None, "--md5", "--no-md5"])
 @pytest.mark.parametrize("explicit_flag", [None, "--explicit"])
 @pytest.mark.parametrize("no_build_flag", [None, "--no-build", "--no-builds"])
 @pytest.mark.parametrize("json_flag", [None, "--json"])
 def test_env_export(
-    export_env, json_flag, no_build_flag, explicit_flag, md5_flag, channel_subdir_flag
+    channel_subdir_flag, md5_flag, explicit_flag, no_build_flag, json_flag, export_env
```
Comment (mathbunnyru) [inline micromamba/tests/test_env.py:98]: This is the order it is mentioned above, it's better to have the same order everywhere

File: micromamba/tests/test_env.py [hunk 2]
```diff
@@ -70,31 +70,47 @@ def export_env():
...
         if md5_flag != "--no-md5":
             assert re.search("#[a-f0-9]{32}$", output.replace("\r", ""))
     else:
-        if json_flag:
-            # Already parsed
-            ret = output
-        else:
-            ret = yaml.safe_load(output)
+        # json is already parsed
+        ret = output if json_flag else yaml.safe_load(output)
```
Comment (mathbunnyru) [inline micromamba/tests/test_env.py:113]: I think this is quite easy to read as a one-line

**Comment by mathbunnyru** [comment]
I think the CI failure is spurious and this can be merged

**Review by jjerphan — APPROVED** [review]
Thank you, @mathbunnyru!

**Pull request merged**